solr/solrj/src/test/org/apache/solr/client/solrj/embedded/LargeVolumeJettyTest.java

```diff
@@ -16,12 +16,17 @@
  */
 package org.apache.solr.client.solrj.embedded;
 
+import java.nio.file.Paths;
+import org.apache.solr.SolrJettyTestBase;
 import org.apache.solr.client.solrj.LargeVolumeTestBase;
 import org.junit.BeforeClass;
 
 public class LargeVolumeJettyTest extends LargeVolumeTestBase {
   @BeforeClass
   public static void beforeTest() throws Exception {
-    initCore();
+    solrClientTestRule
```

Comment:
Let's add a TODO here.  AFAICT, long ago, @gerlowskija removed the Jetty aspect of this test, surely a mistake.

Reply:
🤔 I don't remember doing that in particular, but I guess that'd be expected if it was an unintentional mistake 🤷 

You don't have a commit hash handy do you @dsmiley ?  (No worries if not; I can dig it up with git log/blame, was just hoping to save myself the trouble...)

Reply:
319cb005d314e6f15dd73339e99657d8e39218a4

Reply:
@joshgog in the new Jetty based TestRule you are working on, lets address this issue above (a test that should be using Jetty but isn't any longer) on this test by using the new rule!  Basically, change LargeVolumeTestBase to subclass SolrTestCase instead of EmbeddedSolrServerTestBase, and instead declare whichever rule we want based on the test.

CC @epugh this is the TODO that wasn't clear